# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../qingy/compile.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- qingy-0.9.2/src/libraries/crypto_openssl.c.vanilla	2006-10-24 11:25:23.000000000 +0200
+++ qingy-0.9.2/src/libraries/crypto_openssl.c	2006-10-24 11:25:34.000000000 +0200
@@ -39,6 +39,8 @@
 #include "logger.h"
 
 
+#define QINGY_FAILURE -1
+
 static RSA *rsa = NULL;
 
 
